src/Tools/Console/Command/MappingDescribeCommand.php

```diff
@@ -240,6 +300,22 @@ private function formatValue(mixed $value): string
         throw new InvalidArgumentException(sprintf('Do not know how to format value "%s"', print_r($value, true)));
     }
 
+    /** @throws JsonException */
+    private function formatValueToJson(mixed $value): mixed
```

Comment:
I think `formatValueTo*` should be renamed to `formatValueAs*`, it seems more correct (but I'm not a native english speaker so beware).

Reply:
ahha not native either here ^^ apparently, there's a slight nuance but both are fine, but happy to change it :)